pkgs/development/tools/analysis/hotspot/default.nix

```diff
@@ -64,6 +70,7 @@ mkDerivation rec {
 
   qtWrapperArgs = [
     "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ rustc-demangle ]}"
+    "--prefix PATH : ${lib.makeBinPath [ linuxPackages.perf ]}"
```

Comment:
While we are at adding stuff to `PATH`, maybe we should add binutils too so that the disassembly view works?

Reply:
Wait it doesn't? I guess I tested it in a nix-shell...

Reply:
nix-shell includes stdenv, which includes binutils, so it works there.